chartjs-plugin-annotation.min.js

```diff
@@ -1,10 +1,10 @@
 /*!
  * chartjs-plugin-annotation.js
  * http://chartjs.org/
- * Version: 0.5.3
```

Comment:
Please revert all changes to this file, we will build it during the release process.